querydsl-codegen/src/test/java/com/querydsl/SomeOtherClass2.java

```diff
@@ -1,4 +1,4 @@
-package com.querydsl.core.util;
+package com.querydsl;
```

Comment:
Keep in mind that [`SomeOtherClass` and `SomeOtherClass2` depend on each other](https://github.com/querydsl/querydsl/pull/1313/files#diff-9fe16063e7692477327a759719d8c3a6R6).
The [test loads the `com.querydsl.codegen` package](https://github.com/querydsl/querydsl/pull/1313/files#diff-b8fb56070c211066760fb4bbc68ce563R39) and [verifies if the static block of `SomeOtherClass` was run](https://github.com/querydsl/querydsl/pull/1313/files#diff-b8fb56070c211066760fb4bbc68ce563R41).